src/build.cake

```diff
@@ -153,11 +153,11 @@ Task("Tests")
 		Verbosity = Verbosity.Minimal
 	});
 
-	DotNetTest("./Cody.VisualStudio.Tests/bin/Debug/Cody.VisualStudio.Tests.dll", new DotNetTestSettings 
-    {
-        NoBuild = true,
-        NoRestore = true
-    });
+//	DotNetTest("./Cody.VisualStudio.Tests/bin/Debug/Cody.VisualStudio.Tests.dll", new DotNetTestSettings 

```

Comment:
Do we need to keep this comment?

Reply:
Yes, I will change it later, when I will finish the next part of CI updates.